### PR TITLE
doc: add minutes for meeting 29 June 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2017-06-29.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+* create public nodejs/collaborators repo [#243](https://github.com/nodejs/TSC/issues/243)
+* Rotating meeting times [#198](https://github.com/nodejs/TSC/issues/198)
+
+## Minutes
+
+* TSC Meeting frequency discussion [#286](https://github.com/nodejs/TSC/issues/286)
+  * Agreed to move to monthly meetings.
+  * Agreed to get more input for meeting time.
+* Request to move moderation responsibilities to CommComm [#270](https://github.com/nodejs/TSC/issues/270)
+  * Brian, need to beef up moderation guidelines.  Brian now
```
Comment (MylesBorins) [inline meetings/2017-06-29.md]: s/brian/bryan

File: meetings/2017-06-29.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    thinking that sharing moderation responsibilities between
+    TSC and ComCom. Brian to make suggestions on PR from James as
```
Comment (nebrius) [inline meetings/2017-06-29.md]: nit: we stylize it as CommComm, with two m's :)

File: meetings/2017-06-29.md (hunk 1 above)
Comment (mhdawson) [inline meetings/2017-06-29.md]: Fixed

File: meetings/2017-06-29.md (hunk 2 above)
Comment (mhdawson) [inline meetings/2017-06-29.md]: Fixed

**Comment by mhdawson** [comment]
@MylesBorins @nebrius pushed change to address comments.

**Comment by mhdawson** [comment]
Landed as 2470a8c984159e1dd6e7a33e6c0dde8f9ad8cc92

**Pull request closed without merging**